Resources/Prototypes/_Sunrise/borg_types.yml

```
@@ -26,7 +26,7 @@
   addComponents:
   - type: ShowCriminalRecordIcons
   - type: SiliconLawProvider
-    laws: RobocopLawset
+    laws: SCPDefault #fire edit
```

```suggestion
    laws: SCPDefault # Fire edit
```

Для красоты нужен пробел и большая буква